README.md

```diff
@@ -39,8 +39,16 @@ ln -fs openshift-ansible-*<version>* openshift-ansible
 
 | openshift-ansible url     | OpenShift version | 
 |:-------------------------:|:-----------------:|
+| [3.5.77-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.5.77-1.tar.gz) | OCP 3.5.x |
 | [3.4.50-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.4.60-1.tar.gz) | OCP 3.4 |
 
+* Run `ansible-galaxy` to pull in the necssary requirements for the CASL provisioning:
+
+```
+cd ~/src/casl-ansible
+ansible-galaxy install -r casl-requirements.yml -p roles
+```
+
 Cool! Now you're ready to provision OpenShift clusters
 
 ### Provision a Cluster
```

Comment:
While we're in here, can we add to this section something about the options for setting up DNS? right now, with the galaxy integration we've done, if I run a normal provisioning job that builds a cluster with DNS, then i won't actually exercise the role that was moved over.